_layouts/project.html

```diff
@@ -138,18 +139,31 @@ <h5 class='status-text'>{{ page.status }}</h5>
             {% endif %}
         </ul>
     </div>
+    <!--Project Resources Card-->
+    <div class='page-card-base card-primary page-card-lg page-card--project-description-page'>
+        <h4 class='project-page-card-title'>Resources</h4>
+        <ul class="resource-list unstyled-list">
+            {%- for item in page.links -%}
+                {% if item.name == 'Readme' or item.name == 'Wiki' or item.name == 'Overview' %}
+                    {%- include resource-card.html name=item.name url=item.url image=page.image -%}
+                {% else %}
+                    {%- include resource-card.html name=item.name url=item.url -%}
+                {% endif %}
+            {%- endfor -%}
+        </ul>
+    </div>
 
     <!-- Project Team Card -->
     <div id='project-team-card' class='page-card-base card-primary page-card-lg page-card--project-description-page'>
-        <h4 class='project-page-card-title'>Project Leadership</h4>
+        <h4 class='project-page-card-title'>Current Project Team</h4>
         {% if page.leadership %}
             <div class='resource-list'>
                 {% for item in page.leadership %}
                     <div class='leader-card'>
                         <a href='{{ item.links.github }}' target='_blank' title='GitHub Profile'><img class='leader-img' src='{{ item.picture }}'/></a>
                         <div class='leader-description'>
                             {% if page.status == "Completed" and item.links.linkedin %}
-                                <p class='leader-description-field'><strong>Name: </strong><a href='{{ item.links.linkedin }}' target='_blank' title='Linkedin Profile'>{{ item.name }}</a></p>
+                                <p class='leader-description-field'></p>trong>Name: </strong><a href='{{ item.links.linkedin }}' target='_blank' title='Linkedin Profile'>{{ item.name }}</a></p>
```

Comment:
Overall your code looks great, Evan! I noticed at this line, the `<strong>` tag is missing the opening carrot `<` and an `s`. Can you fix that. 

```suggestion
 <p class='leader-description-field'></p><strong>Name: </strong><a href='{{ item.links.linkedin }}' target='_blank' title='Linkedin Profile'>{{ item.name }}</a></p>
```